yt_dlp/extractor/youtube/_video.py

```diff
@@ -3755,7 +3755,7 @@ def feed_entry(name):
             reason = self._get_text(pemr, 'reason') or get_first(playability_statuses, 'reason')
             subreason = clean_html(self._get_text(pemr, 'subreason') or '')
             if subreason:
-                if subreason == 'The uploader has not made this video available in your country.':
+                if subreason == 'The uploader has not made this video available in your country':
```

Comment:
Nice catch

Reply:
since at one time this was correct with the period, in case YT reverts this change on their end let's make this backwards compatible
```suggestion
                if subreason.startswith('The uploader has not made this video available in your country'):
```